package/base-files/files/etc/profile

```diff
@@ -49,11 +49,22 @@ in order to prevent unauthorized SSH logins.
 EOF
 fi
 
-service() {
-	[ -f "/etc/init.d/$1" ] || {
+service() { 
+	if [ "$1" = "list-enabled" ] && [ -z "$2" ]; then 
+		for F in /etc/init.d/* ; do 
+			$F enabled && echo "$F enabled"
+		done; 
+	elif [ "$1" = "list-disabled" ] && [ -z "$2" ]; then 
+		for F in /etc/init.d/* ; do 
+			$F enabled || echo "$F disabled"
+		done; 
+	elif [ -f "/etc/init.d/$1" ] && [ "$2" = "ubus" ]; then
+		ubus call service list "{'name': '$1'}"
+	elif [ -f "/etc/init.d/$1" ]; then 
+		/etc/init.d/$@
+	else 
 		echo "service "'"'"$1"'"'" not found, the following services are available:"
```

Comment:
echo "service list-enabled|list-disabled|<name start|stop|restart|reload|enable|disable|ubus>" ... etc ...

Reply:
There could be more commands beyond listed, so my fix is slightly different from suggested.